.github/workflows/upload_coverage_for_develop.yml

```diff
@@ -37,4 +37,4 @@ jobs:
 
           # github.event.pull_request.merge_commit_sha is the fresh commit in the develop,
           # provided that github.event.pull_request.merged == true
-          ./codecov -f ./coverage.xml -t ${{ secrets.CODECOV_TOKEN }}  -C ${{ inputs.merge_commit_sha }} -B develop -n "${{ inputs.coverage_artifact_name_in_pr }}"
+          ./codecov -f ./coverage.xml -t ${{ secrets.CODECOV_TOKEN }} -F ONNX -C ${{ inputs.merge_commit_sha }} -B develop -n "${{ inputs.coverage_artifact_name_in_pr }}"
```

Comment:
This is a reusable workflow for both common and ONNX precommit runs, you will need to pass the flag as a parameter. Also don't forget to set up a flag for `COMMON`